eox_hooks/tests/test_actions.py

```diff
@@ -166,7 +166,7 @@ def test_enroll_for_program(self, get_course, create_enroll_task):
         other_course_settings = {
             "EDNX_TRIGGER_FOLLOWUP_ENROLLMENTS": [
                 {
-                    "course_id": "course-v1:edX+DemoX+Demo_Course",
+                    "course_id": "course-v1:edX+Demo123+Demo_Course",
```

Comment:
Is this necessary?

Reply:
Hi @MaferMazu about the test, yes it is necessary, if I leave with DemoX, the test do not run correctly.

And about the configuration in the enrollments, you have to put the same course_id as the parent

Example:

If you put the EDNX_TRIGGER_FOLLOWUP_ENROLLMENTS in the _parent > Other Course Settings_ course-v1:edX+c1+c1, you have to put the same course_id

Like this:
`{
    "EDNX_TRIGGER_FOLLOWUP_ENROLLMENTS": [
        {
            "course_id": "course-v1:edX+c1+c1",
            "mode": "honor"
        },
        {
            "course_id": "course-v1:edX+c1+c1",
            "mode": "honor"
        }
    ]
}`